wolfcrypt/src/port/atmel/atmel.c

```diff
@@ -389,7 +389,7 @@ void atmel_show_rev_info(void)
 #ifdef WOLFSSL_ATECC_DEBUG
     word32 revision = 0;
     atmel_get_rev_info(&revision);
-    printf("ATECC508A Revision: %x\n", (word32)revision);
+    printf("ATECC508A Revision: %x\r\n", (word32)revision);
```

Comment:
Can you rename this to `"ATECC Revision"`? If the `\r` change is important perhaps update the other ones also otherwise please remove.